test/jdk/java/nio/channels/DatagramChannel/NotBound.java

```diff
@@ -77,14 +109,12 @@ public static void main(String[] args) throws IOException {
         // connect
         dc = DatagramChannel.open();
         try {
-            DatagramChannel peer = DatagramChannel.open()
-                .bind(new InetSocketAddress(0));
-            int peerPort = ((InetSocketAddress)(peer.getLocalAddress())).getPort();
-            try {
+            System.out.println("Checks that connect() binds the socket");
```

Comment:
I assume the trace message should be "Check" rather than "Checks" here (same thing for the other messages)